### DISCUSSION TITLE
Need the ability to create user accounts.

### DISCUSSION THREAD

**Discussion by afinkel71** [rhinstaller/anaconda-webui#36] [General]
The new UI is looking nice and easy to follow, but is currently missing the ability to create any user accounts or set a root password. As a result, once the install is complete and the system reboots there isn't a way to login to the newly installed system.  I'm hoping this is planned to be added, but without it now it is difficult to examine the resulting system and see what was installed and how the partitions were created.

**Comment by jkonecny12** [comment]
Hi, yes. Sorry it is a known issue and we are working on resolving that right now. The current ISO has a wrong payload the correct one should be F37 Fedora Workstation like so you would have Gnome Initial Setup which would solve this issue.

Anaconda part for user creation is not yet implemented but we are working on that.

**Reply by jkonecny12** [reply]
The new ISO is there under the same link. It has Wrosktation payload so it's bigger but it's usable after the installation.

**Reply by afinkel71** [reply]
6.9GB seems overly large even for Workstation, but the updated image worked well and successfully installed F37.   On the first reboot I did get presented with the GNOME Initial Setup which let me add a user.

**Reply by jkonecny12** [reply]
Hi, yes the ISO is now almost half of the size (3.8 GB). There were issue with compression of the payload.

**Comment by KKoukiou** [comment]
Feature was added.
